src/pages/[slug].tsx

```diff
@@ -33,6 +36,7 @@ export async function getServerSideProps({
       props: {
         config: getFallbackConfig() as unknown as ConfigType,
         configName,
+        packageAttributes: getServiceSidePackageAttributes(),
```

Comment:
I would suggest moving this definition to the `App.getInitialProps` in the `_app.tsx`. Then it would be initialized centralized and no code duplication would be present.